### ISSUE TITLE
Colour & Contrast Balance

### ISSUE THREAD

**Issue by StarWitch** [taisei-project/taisei#198]
I had someone who had never played Taisei before take a crack at the game. One the major issues they came across was colour and contrast balance.

The game-wide issues were:
1. Player shots (Marisa's lasers, Reimu's ofuda, even Youmu's standard shots) were too bright and visually "noisy." I've done some experiments reducing the opacity by 60% and I have to say it does make things a lot more readable.
2. Shades of red and blue between Powerups and Points are too similar to red/blue danmaku, and actually look like some of the enemy "tag" danmaku that comes up. They're just not visually distinct enough. (More transparency on these, too?)
3. Backgrounds aren't balanced against bullets either. Some stages are blindingly bright (Stage 1 and 3 come to mind) but it also comes into play when boss spellcards enable. Elly's final spellcard is difficult to read at the best of times because the formulas on the background behind her are the same brightness as the danmaku flying at you.
4. Bullets could stand to be more vivid and with stronger borders. Some examples: the ice crystals in Stage 1 tend to blend in with the water, Iku's violet "thunder laser" in Stage 5 needs to stand out more, and some of the orbs in Stage 6 feel "fuzzier" than they ought to be. 

So, the plan ought to be:
1. Increase player shot transparency
2. Increase powerup/point transparency
3. Darken backgrounds and/or "tint" them so they're visually distinct from the danmaku in each stage
4. Make danmaku more distinct and vivid

**Comment by HybridDog** [comment]
Maybe it is possible to make the collectibles unsharp and less saturated when they are near the enemy projectiles. I sometimes find it difficult to distinguish between both. 

Anyways, here are two screenshots:
![fot_2020-04-10-20:55:35_744430367](https://user-images.githubusercontent.com/3192173/79016452-49ebd500-7b6f-11ea-9a81-a5552f03a6bf.jpg)
![fot_2020-04-10-21:04:17_892039858](https://user-images.githubusercontent.com/3192173/79016776-1bbac500-7b70-11ea-86d6-fe9c2ac619e1.jpg)

**Comment by Akaricchi** [comment]
The stage 3 backgound is indeed getting replaced to fit the new story better (as well as to solve the brightness problem)

**Comment by StarWitch** [comment]
Yeah, the lack of visual distinction between Point/Power pickups and bullets is definitely an issue that came up with my playtesters.
